src/index.js

```diff
@@ -0,0 +1,52 @@
+import './style.css';
+import UpdateStorage from './Modules/updateStorage.js';
+import { UpdateTask, saveUpdatedTask } from './Modules/edit.js';
+import {
+  displayTasks, deleteTask, add, gameChange, getTodos, demolishAll,
+} from './functions.js';
+
+// variables
+// let list = JSON.parse(localStorage.getItem('storedStTask')) || [];
```

Comment:
- [ ] try as much as possible to avoid leaving unused codes as comments in your codebase since it makes your code bulky and slow